lib/inc/drogon/HttpBinder.h

```diff
@@ -173,7 +173,7 @@ class HttpBinderBase
 template <typename T>
 T &getControllerObj()
 {
-    // Initialization of function-local statics is guaranteed to occur only once
+    // Initialization of function-local statistics is guaranteed to occur only once
```

Comment:
Initialization of function local static variables is guaranteed to occur only once

Reply:
```suggestion
    // Initialization of function-local statics is guaranteed to occur only once
```